wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/impl/ProjectImportManager.java

```diff
@@ -318,8 +319,7 @@ private RegisteredProject doImportInternally(
         projectConfigRegistry.put(newProjectConfig, true, false);
       }
 
-      return projectConfigRegistry
-          .get(wsPath)
+      return Optional.ofNullable(projectConfigRegistry.getOrNull(wsPath))
```

Comment:
You can use `return projectConfigRegistry.get(wsPath)` here if I'm not mistaken, but up to you

Reply:
@RomanNikitenko I guess you are right, just considered it as risky at that time :)
if you or @dkuleshov think it is totally safe - will fix it immediatelly
WDYT?

Reply:
I guess we can safely use `projectConfigRegistry.get(wsPath)` since it already returns `optional`.

Reply:
ok, done